SiemensCommunity-API/Service/Implementations/AccountService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Data.Contracts;
+using Microsoft.AspNetCore.Identity.UI.Services;
+using Service.Adapters;
+using Service.Contracts;
+using Service.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Service.Implementations
+{
+    public class AccountService : IAccountService
+    {
+        private readonly IAccountRepository _accountReposistory;
+        private readonly UserAdapter _userAdapter = new UserAdapter();
+        private readonly ForgotPasswordAdapter _forgotPasswordAdapter = new ForgotPasswordAdapter();
+        private readonly IEmailService _emailService;
+
+        public AccountService(IAccountRepository accountReposistory, IEmailService emailService)
+        {
+            _accountReposistory = accountReposistory;
+            _emailService = emailService;
+        }
+
+        public async Task<int> RegisterAsync(UserRegisterCredentials userCredentials)
+        {
+            var returnedUser = await _accountReposistory.RegisterAsync(_userAdapter.AdaptToUserIdentity(userCredentials), userCredentials.Password);
+            return returnedUser;
```

Comment:
Return type is Task<int> but the name of the returned variable is "returnedUser"?